spec/sequel/plugins/enum_values_spec.rb

```diff
@@ -229,6 +229,62 @@
 			end
 		end
 	end
+
+	describe 'Sequel::Model.enum_filter' do
+		subject(:enum_filter) { item_class.public_send(enum_value).sql }
+
+		let(:filter_methods) { true }
```

Comment:
I want to see tests of default state, like without this option. You can look at https://github.com/AlexWayfer/sequel-enum_values/blob/6227605dcc3d62fff8adc3617e64d76e6f44aa28/spec/sequel/plugins/enum_values_spec.rb#L176-L180, for example.

Reply:
Added tests